src/Parser/Serialization/DefaultSerializer.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Parser
+{
+
+    ///<summary>TODO combine with NLog's Json (injected) serializer?</summary>
```

Comment:
agree, but then we need to have:

a) a dependency to NLog (back), so we need to merge it with NLog
b)  or a we have to always to inject the ISerializer, which makes this library as less working out of the box. 
c)  or we need to extract the serializer to a new package

What do you think is the best?